### PR TITLE
Fixed some Labeling issues in Xtext UI Contributions

### PR DESCRIPTION
**Pull request by cdietrich**
https://bugs.eclipse.org/bugs/show_bug.cgi?id=420434
https://bugs.eclipse.org/bugs/show_bug.cgi?id=420435

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.ui.shared/plugin.properties [hunk 1]
```diff
@@ -2,6 +2,7 @@ OpenModelElementCommand.name=Open Model Element
 OpenModelElementCommand.description=Open the declaration of an Xtext element
 XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
+XtextNature.description=Adds an icon decoration to projects with Xtext enabled.
```
Comment (LorenzoBettini) [inline org.eclipse.xtext.ui.shared/plugin.properties]: shouldn't this be something like "Convert the project to an Xtext project"? I mean, adding a nature is more than adding an icon decoration

Comment (cdietrich) [inline org.eclipse.xtext.ui.shared/plugin.properties]: no this description is about the icon

Comment (cdietrich) [inline org.eclipse.xtext.ui.shared/plugin.properties]: (i took the text from the maven decorator icon)

**Pull request merged**